packages/app/src/cli/utilities/extensions/name-mapper.ts

```diff
@@ -0,0 +1,84 @@
+import {ExtensionTypes, ExtensionTypeslKeys, UIExtensionTypes, UIExtensionTypeslKeys} from '../../constants'
+
+export function convertExtensionTypesToExtensionTypeKeys(extensionTypes: ExtensionTypes[]): ExtensionTypeslKeys[] {
+  return extensionTypes.map(convertExtensionTypeToExtensionTypeKey)
```

Comment:
It is only a wrapper utility method because this mapping is used in several places. Besides it could help in future uses oriented to functional programming.